Update documentation about configuring expo versions (#1674)
microsoft/vscode-react-native: README.md

## Configure dependencies versions for debugging Expo projects

> To debug Expo applications the extension requires additional dependencies, such as `xdl` and `@expo/metro-config`. These packages are installed automatically on the first start of the Expo debugging scenario. You can configure the dependencies versions by setting the parameter to the appropriate package name in the `react-native.expo.dependencies` object in `settings.json`:

Reviewer: ```suggestion
To debug Expo applications the extension requires additional dependencies, such as `xdl` and `@expo/metro-config`. These packages are installed automatically on the first start of the Expo debugging scenario. You can configure the dependencies versions by adding the `react-native.expo.dependencies` parameter, which contains packages names and appropriate versions, to the `settings.json` file:
```

```json
{
  "react-native.expo.dependencies": {
    "xdl": "59.1.0",

>     "@expo/metro-config": "0.1.72"

Reviewer: ```suggestion
    "@expo/metro-config": "0.1.72",
    "@expo/ngrok": "4.1.0"
```
